packages/design-system/src/styles/components/_HelpDrawer.scss

```diff
@@ -72,5 +97,21 @@
 }
 
 .ds-c-help-drawer__footer {
+  background-color: $help-drawer-footer-fill;
   flex-shrink: 0;
+  font-size: $small-font-size;
+  line-height: 1.5;
+  padding: $help-drawer-padding;
+
+  @media (min-width: $width-lg) {
+    font-size: $base-font-size;
```

Comment:
It looks like the media queries that you replaced actually applied to the `help-drawer__content` and not the footer. I can see the difference when I look at it on small screens:

![footer-size](https://user-images.githubusercontent.com/7595652/135931765-09be28a7-5d7f-490b-9fd5-4671f6cb8511.gif)

Reply:
So it is... wonder why VisReg didn't pick it up 🤔